### PR TITLE
Project Creation Tutorial

### PR DESCRIPTION
**Pull request by ryannk**
Added a tutorial on how to create a project with pictures

### CODE REVIEW

File: learner/tutorials/eclipse.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+    Server server = new Server(5757, new ServerPreferences(10000, 50), annotator);
+
+    // We have no more work to do, so we will use the executeInstance method to start and keep our Server alive
+    fi.iki.elonen.util.ServerRunner.executeInstance(server);
+}
+```
+
+You should now be able to compile and run this program. If it is successful it should print "Your learner endpoint is listening at this address..."
+
+You can press enter in the terminal to end the program.
```
Comment (paultgibbons) [inline learner/tutorials/eclipse.md]: Is this what we want?

File: learner/tutorials/eclipse.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+Once you have created your `Server` you can run it using `ServerRunner.executeInstance`. Code doing the that with our `ToyPosAnnotator` is shown below.
+
+```java
+public static void main(String args[]) throws IOException {
+    // Create the annotator
+    Annotator annotator = new ToyPosAnnotator();
+
+    // We will have our server listen on port 5757 and pass it our toy annotator
+    // Tell the client to request batches with 50 instances
+    Server server = new Server(5757, new ServerPreferences(10000, 50), annotator);
```
Comment (paultgibbons) [inline learner/tutorials/eclipse.md]: 10000 should be explained - also should say that 50 is a cap

File: learner/tutorials/eclipse.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+}
+```
+
+You should now be able to compile and run this program. If it is successful it should print "Your learner endpoint is listening at this address..."
+
+You can press enter in the terminal to end the program.
+
+## Part 5: Run the learner endpoint with the Open Eval system
+
+Start your learner endpoint. Open a browser and go to the Open Eval site. Log in or create an account if needed. Create a configuration with your desired dataset and task. For our example it is "500" and "POS".
```
Comment (paultgibbons) [inline learner/tutorials/eclipse.md]: Also tell them to turn off their firewall

File: learner/tutorials/eclipse.md (hunk 2 above)
Comment (ryannk) [inline learner/tutorials/eclipse.md]: I explain that 50 is recommended above

File: learner/tutorials/eclipse.md (hunk 1 above)
Comment (ryannk) [inline learner/tutorials/eclipse.md]: What would the alternative be?

Comment (paultgibbons) [inline learner/tutorials/eclipse.md]: ctrl c ?

Comment (ryannk) [inline learner/tutorials/eclipse.md]: That also currently works

**Comment by danyaljj** [comment]
wow ok!!

File: learner/tutorials/eclipse.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
+# Create a Open Eval client project using Eclipse
```
Comment (danyaljj) [inline learner/tutorials/eclipse.md]: Do you think you can link to this readme from outside? Like from `learner/README.md` maybe?

Comment (ryannk) [inline learner/tutorials/eclipse.md]: Just included it at the top in italics

Comment (danyaljj) [inline learner/tutorials/eclipse.md]: cool!

**Pull request merged**